src/initial_conditions/update_initial_conditions.jl

```diff
@@ -1,4 +1,4 @@
-function update_initial_conditions(
+function update_initial_conditions!(
```

Comment:
Not sure if no `!` was intentional, but it felt wrong to me. Tell me if I should revert these changes.